ehrql/measures/measures.py

```diff
@@ -300,6 +306,31 @@ def configure_dummy_data(self, *, population_size):
         """
         self.dummy_data_config.population_size = population_size
 
+    def configure_disclosure_control(self, *, enabled=True):
+        """
+        Configure disclosure control.
+
+        By default, the numerators and denominators of measures generated from real and
+        dummy tables, and by the dummy data generator, are subject to disclosure
+        control. First, values less than or equal to `7` are replaced with `0`
+        (suppressed); then, values are rounded to the nearest `5`.
+
+        To disable disclosure control in such cases, then:
+
+        ```py
+        measures.configure_disclosure_control(enabled=False)
+        ```
+
+        Measures provided by a dummy data file are not subject to disclosure control.
+
+        For more information about disclosure control in OpenSAFELY, please see the
+        "[Updated disclosure control
+        guidance](https://www.opensafely.org/updated-output-checking-processes/)" page.
+        """
+        # The following is covered by tests/integration/test_main.py, but coverage can't
+        # detect that.
+        self.disclosure_control_config.enabled = enabled  # pragma: no cover
```

Comment:
It will be because the definition file is evaluated in a different process (for sandboxing purposes).

I think it might make sense to have a unit test for this anyway in `tests/unit/measures/test_measures.py`. It's obviously pretty trivial, but if we start configuring more stuff here and needing some kind of validation logic then it would be good to have a pre-existing test to expand on. And I feel like having the test is simpler than explaining the presence of the pragma.

Reply:
That's correct.

I've added `test_configure_disclosure_control`. I've also updated `test_define_measures` and `test_define_measures_with_default_group_by` to test for the default value.